packages/block-library/src/gallery/gallery-image.js

```diff
@@ -128,6 +128,25 @@ class GalleryImage extends Component {
 		/* eslint-disable jsx-a11y/no-noninteractive-element-interactions, jsx-a11y/onclick-has-role, jsx-a11y/click-events-have-key-events */
 		return (
 			<figure className={ className } tabIndex="-1" onKeyDown={ this.onKeyDown } ref={ this.bindContainer }>
+				{ process.env.GUTENBERG_PHASE === 2 ?
+					isSelected && <div className="block-library-gallery-item__move-menu">
+						<IconButton
+							icon="arrow-left"
+							onClick={ onMoveBackward }
+							className="blocks-gallery-item__move-backward"
+							label={ __( 'Move Image Backward' ) }
+							aria-disabled={ isFirstItem }
+						/>
+						<IconButton
+							icon="arrow-right"
+							onClick={ onMoveForward }
```

Comment:
Should we do `isLastItem ? undefined : onMoveForward` to be consistent with the `aria-disabled` prop regardless of the check present in the parent component?

Reply:
It doesn't harm 63e0bf0